2 - ListsLogicAndIterations/Code/Arrays.cs

```diff
@@ -1,17 +1,27 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace ListsLogicAndIterations.Code
 {
 	public static class Arrays
 	{
-		public static int[] BuildIntArray(int fist, int second)
+    
+		public static int[] BuildIntArray(int first, int second)
 		{
-			throw new NotImplementedException();
+            int[] array1 = new int[] { first, second };
+            return array1;
```

Comment:
What you did is correct. If you wanted to, you could have have used 'var' instead of `int[] array1 ...`. You could also turn this into a single line `return new int[] {first, second};`.

This "type" of comment of using var and directly returning you can apply to other places of your code but I won't bloat this code review. Just something to note that you *can* do going forward but not something you *must* do - just telling you whats possible.